mcp_health_check.py

```
@@ -0,0 +1,1246 @@
+                    with urllib.request.urlopen(req, timeout=10) as response:  # nosec B310
+                        result = CheckResult(
+                            name="WooCommerce API Connectivity",
+                            status=CheckStatus.OK,
+                            message=f"Connected to WooCommerce at {wc_url}",
+                        )
+                except urllib.error.HTTPError as e:
+                    if e.code in (401, 403):
+                        result = CheckResult(
+                            name="WooCommerce API Connectivity",
+                            status=CheckStatus.OK,
+                            message=f"WooCommerce API reachable (auth required)",
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)